cupy/core/include/cupy/complex/complex.h

```diff
@@ -62,7 +62,12 @@ struct _select_greater_type
  *
  */
 template <typename T>
+#if defined(__CUDACC__)
```

Comment:
Can you also test `__CUDACC_RDC__` here? It's a preparation for #2426.

Reply:
I think both `__CUDACC__` and `__CUDACC_RDC__` are set when built with `--relocatable-device-code=true`.

Reply:
Ah, I see! Yes, I can verify. Thanks @kmaehashi! This is not obvious from the documentation...☹️